.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug report, needs-triage
+assignees: ''
+
+---
+
+Confirm by changing [ ] to [x] below to ensure that it's a bug:
+- [ ] I've searched for [previous similar issues](https://github.com/aws/aws-iot-device-sdk-js-v2/issues) and didn't find any solution
+  
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**SDK version number**
+
```

Comment:
As with other SDKs, we need Platform/OS/Hardware info as well even when it's a managed language.